doc/source/ray-air/examples/batch_tuning.ipynb

```diff
@@ -0,0 +1,1801 @@
+    "\n",
+    "To demonstrate how batch training can be parallelized, we will train a separate model for each dropoff location. This means we can use the `dropoff_location_id` column in the dataset to group the dataset into data batches. Then we will fit a separate model for each batch and evaluate it."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f886edf7",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "# Contents\n",
+    "\n",
+    "In this this tutorial, you will learn about:\n",
+    " 1. [Introduction to Ray Tune](#intro_tune)\n",
+    " 2. [Define how to load and prepare Parquet data](#load_data)\n",
+    " 3. [Define your Ray Tune Search Space and Search Algorithm](#define_search_space)\n",
+    " 4. [Define a Trainable (callable) function](#define_trainable)\n",
+    " 5. [Run batch training on Ray Tune](#run_tune_search)\n",
+    " 6. [Load a model from checkpoint and perform inference](#load_checkpoint)\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2d70a863",
+   "metadata": {},
+   "source": [
+    "# Introduction to Ray Tune <a class=\"anchor\" id=\"intro_tune\"></a>\n",
+    "\n",
+    "**While Tune's main purpose is hyperparameter optimization, you can also use it as an execution engine to run independent parallel jobs.**\n",
+    "\n",
+    "In this notebook, we will use [Ray Tune](https://docs.ray.io/en/latest/tune/index.html) to conduct batch tuning by evaluating several models for each dropoff location. After the Tune experiment (all the trials) has finished, we will pick the best model per dropoff location.\n",
+    "\n",
+    "```{note}\n",
+    "An experiment in Tune is defined as a set of trials.\n",
+    "```\n",
+    "\n",
+    "Let us quickly walk through the [key concepts](https://docs.ray.io/en/latest/tune/key-concepts.html) you need to know to use Tune.\n",
+    "\n",
+    "- First, you define a *Search space* and pass that into a `Trainable` or `callable` function, that specifies the objective you want to tune.  The trainable function will be called for every trial in the search space.\n",
+    "\n",
+    "- Then you select a *Search algorithm* and optionally use a\n",
+    "scheduler to stop searches early and speed up your experiment.  We will use simple **grid search** (run every permutation as a separate trial).\n",
+    "\n",
+    "- Together with other configurations, the Trainable, Search algorithm, and Scheduler are passed into a [`Tuner`](air-tuner-ref), which runs your experiment trials in parallel.\n",
+    "\n",
+    "- The output from `Tuner.fit()` can be analyzed to inspect your experiment results.  The following figure shows an overview of the Ray Tune flow, which we will use in this tutorial.\n",
+    "\n",
+    "![Tune flow diagram](../../tune/images/tune_flow.png)"
+   ]
+  },
```

Comment:
Instead of this, we should just have caption/tooltip that says "Prerequisite for this notebook: Read the Key Concepts page for Ray Tune".